x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -445,11 +453,20 @@ export class Plugin implements ISecuritySolutionPlugin {
 
     setIsElasticCloudDeployment(plugins.cloud.isCloudEnabled ?? false);
 
+    this.telemetryEventsSenderV2.setup(
+      DEFAULT_RETRY_CONFIG,
```

Comment:
From an API perspective: it seems a bit odd that the configuration is defined on the instance here, while the channel is specified on each `send` method call. Maybe I was reading too much into your example's emphasis on per-channel configuration, but I don't see a nice way to do something like that without creating multiple instances and/or updating configuration on the fly.

Is there a workflow that this particular orientation enables?